cabot/metricsapp/forms/grafana_elastic.py

```diff
@@ -1,29 +1,44 @@
 from django import forms
 import json
+
+from cabot.cabotapp.utils import group_fields
 from .grafana import GrafanaStatusCheckForm, GrafanaStatusCheckUpdateForm
 from cabot.metricsapp.models import ElasticsearchStatusCheck
 
+_FIELDS = [
```

Comment:
I wonder if instead of repeating field names in the `_FIELDS` and `_GROUPS` variables, we should write them once in a nested list, eg:

```
_GROUPED_FIELDS = [
    '[Basic': ['name', 'active', 'service_set']],
    '[Thresholds': ['check_type', 'warning_value', 'high_alert_importance', 'high_alert_value']],
    ...
]
```

And then later:

```
fields = get_fields_from_groups(_GROUPED_FIELDS)  # Additional utility function?
```

Would require a little rework but would reduce repetition.  WDYT?